app.rb

```diff
@@ -0,0 +1,103 @@
+require './student'
+require './classroom'
+require './person'
+require './rental'
+require './book'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No people available'
+    else
+      @people.each { |people| puts "[#{people.class}] Name: #{people.name}, ID: #{people.id}, Age: #{people.age}" }
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2) [Input the number]:'
+    option = gets.chomp.to_i
+    case option
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def create_student
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp
+    puts 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+    @people << (Student.new(age, name, parent_permission))
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp
+    puts 'specialization: '
+    specialization = gets.chomp
+    @people << (Teacher.new(age, name, specialization))
+    puts 'Person created successfully'
```

Comment:
- [ ] Well done creating the UI everything was working well till I tries to create a teacher, when I choose a specialization the I've got the following error message. Kindly make sure to fix that issue.


<img width="653" alt="Screenshot 2023-10-05 at 9 20 29 PM" src="https://github.com/JavierAybar/OOP-school-library/assets/53347966/4ef78068-6b1a-4b71-8d9d-d9e8f7a2377b">